Relevant test failure:

```
shakenfist_ci.tests.test_state_changes.TestStateChanges.test_lifecycle_power_cycle
----------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/home/ubuntu/shakenfist/deploy/shakenfist_ci/tests/test_state_changes.py", line 134, in test_lifecycle_power_cycle
    self._await_instance_ready(inst['uuid'])

      File "/home/ubuntu/shakenfist/deploy/shakenfist_ci/base.py", line 127, in _await_instance_ready
    self._await_agent_state(instance_uuid, ready=True)

      File "/home/ubuntu/shakenfist/deploy/shakenfist_ci/base.py", line 152, in _await_agent_state
    raise TimeoutException(

    shakenfist_ci.base.TimeoutException: Instance 1da6e6de-e0d7-4efb-b9a9-e14ba8d0970b failed to start
    and enter the agent ready state in five minutes. Agent state is not ready (instance powered off).
```

--- a/shakenfist/daemons/sidechannel.py
+++ b/shakenfist/daemons/sidechannel.py
@@ -145,7 +145,8 @@ class Monitor(daemon.Daemon):
                         except (BrokenPipeError,
                                 ConnectionRefusedError,
                                 ConnectionResetError,
-                                FileNotFoundError):
+                                FileNotFoundError,
+                                OSError):
                             if sc in sc_clients:
                                 del sc_clients[sc]
 

